overviewer_core/textures.py

```diff
@@ -2167,13 +2167,41 @@ def crops(self, blockid, data):
     alpha_over(img, crop3, (6,3), crop3)
     return img
 
-# farmland
-@material(blockid=60, data=range(9), solid=True)
-def farmland(self, blockid, data):
-    top = self.load_image_texture("assets/minecraft/textures/blocks/farmland_wet.png")
-    if data == 0:
-        top = self.load_image_texture("assets/minecraft/textures/blocks/farmland_dry.png")
-    return self.build_block(top, self.load_image_texture("assets/minecraft/textures/blocks/dirt.png"))
+# farmland and grass path (15/16 blocks)
+@material(blockid=[60,208], data=range(9), solid=True)
+def farmland_and_grass_path(self, blockid, data):
+    if blockid=60:
```

Comment:
You probably mean `if blockid == 60` here. Other than that, I can't see anything obviously out of place.